assets/lambda/code/scan/lambda.py

```diff
@@ -104,15 +104,24 @@ def lambda_handler(event, context):
 
 
 def set_status(bucket, key, status):
+    """Get previous tags and append scan-status tag"""
```

Comment:
```suggestion
    """Set the scan-status tag to the S3 Object"""
```
The doc string gives a general idea of the function, I don't think we need to explain all the steps

Reply:
updated